plugins/dfu/dfu-device.c

```diff
@@ -1216,6 +1216,7 @@ dfu_device_open (FuUsbDevice *device, GError **error)
 	DfuDevice *self = DFU_DEVICE (device);
 	DfuDevicePrivate *priv = GET_PRIVATE (self);
 	GPtrArray *targets = dfu_device_get_targets (self);
+	GUsbDevice *usb_device = fu_usb_device_get_dev (device);
```

Comment:
this seems it should go into the scope of the `if` loop

Reply:
Ack, moved.